third-party/realdds/src/dds-device-impl.h

```diff
@@ -34,19 +34,32 @@ class flexible_msg;
 
 class dds_device::impl
 {
+    enum class state_t
```

Comment:
Rename `state`. Other types we declare don't have a `_t` suffix.

Reply:
Yeah, I like it too but, since we use all-lowercase, you do get in trouble with variables named `state`, which we use.
So, just like `size_t`, I made it `state_t` to make sure it's not conflicting. I think minor, no?

Reply:
I did not see any `state` that might conflict here, but it is minor.